presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSplitManager.java

```diff
@@ -64,6 +66,9 @@ public ConnectorSplitSource getSplits(
             return new FixedSplitSource(ImmutableList.of());
         }
 
+        TupleDomain<IcebergColumnHandle> predicate = isPushdownFilterEnabled(session) ?
+                layoutHandle.getPartitionColumnPredicate().transform(IcebergColumnHandle.class::cast) : table.getPredicate();
```

Comment:
nit: put table.getPredicate(); on a separate line

Reply:
@imjalpreet I took the liberty to resolve this and put it on a separate line myself.

Reply:
Yes sure, thank you. I will fix the checkstyle issue and squash that commit as well.